samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/configuration/WebSecurityConfig.java

```diff
@@ -127,11 +125,12 @@ public void configure(HttpSecurity http) throws Exception {
         .logout().logoutSuccessUrl("/login.html").and()
 
         // register login and logout filter that handles rest logins
+        .addFilterBefore(basicAuthenticationFilter(), BasicAuthenticationFilter.class)
```

Comment:
This is a nice solution to make the tests work easily. However, we should discuss how to further proceed. Should this be only enabled in test mode? As we have to revisit the entire authentication mechanism for OASP I would also accept this as is and file an issue for the general rework of the OASP authentication.